recipes/install-admin.rb

```diff
@@ -0,0 +1,30 @@
+# Extract zip then move the war file to the tomcat directory
+aps_admin_version = node['aps-core']['admin_app']['version']
+admin_version = node['aps-core']['admin_app']['version']
+zip_location = "#{Chef::Config[:file_cache_path]}/activiti-admin-#{aps_admin_version}.zip"
+tomcat_home = node['aps-core']['appserver']['tomcat_home']
+war_location = "/tmp/activiti-admin-#{admin_version}/activiti-admin/activiti-admin.war"
+
+package 'unzip'
+
+execute "unzip -q -u -o #{zip_location} -d /tmp" do
+  not_if { Dir.exist?("/tmp/activiti-admin-#{admin_version}") }
+  only_if { File.exist?(zip_location) }
+end
+
+package 'unzip' do
```

Comment:
any reason why we remove it?
If this is part of the redeploy and we call the redeploy several times (for update) we'd install it every time

Reply:
It was just for tidying up. Its not part of the redeploy so it wont matter